android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -925,6 +925,12 @@ public void onHostDestroy() {
       cleanupCallbacksAndDestroy();
     }
 
+    // Prevent HLS audio from going to sleep when app goes into background
+    @Override
+    protected void onWindowVisibilityChanged(int visibility) {
+      if (visibility != View.GONE) super.onWindowVisibilityChanged(View.VISIBLE);
```

Comment:
Is this actually trying to check if the view is `INVISIBLE` and if it is then set it to `VISIBLE`?

Reply:
@jayfunk `INVISIBLE` is different from `GONE` since it takes up space on the screen. It's similar to `visibility: hidden` vs `display: none`.

See https://developer.android.com/reference/android/view/View#GONE and https://developer.android.com/reference/android/view/View#INVISIBLE